pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/JdbiCompositeQuestion.java

```diff
@@ -27,6 +27,10 @@ default void insertChildren(long compositeQuestionId, List<Long> childQuestionId
         insertChild(compositeQuestionId, childQuestionIds, IntStream.rangeClosed(0, childQuestionIds.size() - 1).boxed().collect(toList()));
     }
 
+    default void insertChildren(long compositeQuestionId, List<Long> childQuestionIds, List<Integer> orderIdxs) {
```

Comment:
I could have done this update in StudyBuilder task but added to core for any future use. Nothing other than new SB task uses this, so no additional test impact